content/docs/command-reference/remote/add.md

```diff
@@ -61,6 +61,9 @@ DVC will determine the [type of remote](#supported-storage-types) based on the
 - `--system` - save remote configuration to the system config (e.g.
   `/etc/xdg/dvc/config`) instead of `.dvc/config`.
 
+- `--project` - save remote configuration to the project's config
+  (`.dvc/config`). Used by default.
+
```

Comment:
Q @efiop: if this is used by default and `remote add` only writes why even have the option here? When does it have an effect?

Reply:
Same for `remote modify/remove/rename`

Reply:
For symmetry mostly. Right now remtoe add/modify/remove/rename will just default to --project anyway.

Reply:
Thanks @efiop. So just to be clear, the option in these commands never has an effect right?

Reply:
Yes, it is the default behavior.